src/handler.js

```diff
@@ -114,11 +116,45 @@ const handle = async message => {
       Authorization: `Bearer ${process.env.ACCESS_TOKEN}`,
     },
   }).then(response => {
-    response.data.pipe(resizer).pipe(writeStream);
+    response.data.pipe(resizer).pipe(resizedStream);
+  });
+
+  let b64Chunks = [];
+
+  resizedStream.pipe(b64Stream);
+
+  b64Stream.pipe(new Base64Encode());
+
+  b64Stream.on('data', chunk => {
+    b64Chunks.push(chunk);
+  });
+  const [owner, repo] = process.env.GITHUB_REPO.split('/');
+  const branch = process.env.GITHUB_REPO_BRANCH;
+  const emojiRepoDir = process.env.GITHUB_REPO_DIR;
+  const emojiPath = path.join(emojiRepoDir, `${emojiAlias}.${filetype}`);
+
+  b64Stream.on('finish', async () => {
+    console.log('b64', Buffer.concat(b64Chunks).toString('base64'));
+    try {
+      await octokit.repos.createFile({
+        owner,
+        repo,
+        path: emojiPath,
+        branch,
+        message: `emojo: added :${emojiAlias}:`,
+        content: Buffer.concat(b64Chunks).toString('base64'),
+      });
+      logger.info(`Created ${emojiAlias}.${filetype} in ${owner}/${repo}`);
+    } catch (e) {
+      logger.error(
+        `Error creating file on Github ${owner}/${repo}/${emojiPath}`,
+        e
+      );
+    }
   });
 
   const streamAsPromise = new Promise((resolve, reject) =>
-    writeStream.on('finish', resolve).on('error', reject)
+    resizedStream.on('finish', resolve).on('error', reject)
   );
 
   return streamAsPromise.then(async () => {
```

Comment:
You'll need to refactor this bit probably to use promise.all. At the moment you're waiting for the first stream to finish and the returning a response body to the Lambda handler but the b64 one might not have finished yet. Need to only return when resizedStream finishes and b64stream finsihes